### PR TITLE
Upgrade KubeVirt to v1.1.0

### PR DESCRIPTION
**Pull request by atanasdinov**
- Upgrades KubeVirt operator from v0.58.0 to v1.1.0
- Drops CDI dependency
- Upgrades package version to v0.2.0 in order to avoid versioning collisions - v0.2.0 will eventually be the stable we use for Edge 3.0

### CODE REVIEW

**Comment by atanasdinov** [comment]
Functionality of the chart is kept as is (aside from dropping the CDI dependency). Incoming PRs will address:
- hooks using a separate namespace
- CDI upgrade to v1.57
- separate CDI chart
- hardening options and Sylva integration adjustments as requested in https://github.com/suse-edge/charts/issues/50
- more customization options

File: index.html [hunk 1]
```diff
@@ -201,14 +201,14 @@ <h2>Charts</h2>
 			
         
           <div class="chart">
-            <a href="assets/kubevirt/kubevirt-0.1.0.tgz" title="assets/kubevirt/kubevirt-0.1.0.tgz">
+            <a href="assets/kubevirt/kubevirt-0.2.0.tgz" title="assets/kubevirt/kubevirt-0.2.0.tgz">
               <div class="icon">
                 <img class="chart-item-logo" alt="kubevirt's logo" src="https://github.com/cncf/artwork/raw/master/projects/kubevirt/icon/color/kubevirt-icon-color.svg">
```
Comment (e-minguez) [inline index.html:206]: If you can fix this as well it would be nice :)

File: index.yaml [hunk 2]
```diff
@@ -47,6 +47,17 @@ entries:
     - assets/endpoint-copier-operator/endpoint-copier-operator-0.2.0.tgz
     version: 0.2.0
   kubevirt:
+  - apiVersion: v2
+    appVersion: 1.1.0
+    created: "2023-12-04T18:24:29.017707+02:00"
+    description: A Helm chart for KubeVirt
+    digest: bfbd0cb5410c72fbac2cfee6d43d270b845b694a114fe1bafbd862615a1d67e9
+    icon: https://github.com/cncf/artwork/raw/master/projects/kubevirt/icon/color/kubevirt-icon-color.svg
```
Comment (e-minguez) [inline index.yaml:55]: Same

File: charts/kubevirt/0.2.0/values.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+kubevirt:
+  image: registry.suse.com/suse/sles/15.5/virt-operator:1.1.0-150500.8.6.1
+  version: 1.1.0-150500.8.6.1
+  pullPolicy: IfNotPresent
+
+kubecli:
+  image: bitnami/kubectl:1.27.3
```
Comment (e-minguez) [inline charts/kubevirt/0.2.0/values.yaml:7]: Ugh. If this is really needed, maybe we can just use rancher shell? https://github.com/rancher/shell

**Review by e-minguez — CHANGES_REQUESTED** [review]
Some nits :)

File: charts/kubevirt/0.2.0/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+appVersion: 1.1.0
+description: A Helm chart for KubeVirt
+icon: https://github.com/cncf/artwork/raw/master/projects/kubevirt/icon/color/kubevirt-icon-color.svg
```
[the remark this answers is not in the document]
Comment (atanasdinov) [reply charts/kubevirt/0.2.0/Chart.yaml:4]: Guilty. I saw https://github.com/suse-edge/charts/pull/22 and didn't even bother to click on it. 😄

File: charts/kubevirt/0.2.0/values.yaml (hunk 3 above)
Comment (atanasdinov) [reply charts/kubevirt/0.2.0/values.yaml:7]: I'll try this one in a follow up once we merge this and v0.2.0 is accessible in Rancher.

File: charts/kubevirt/0.2.0/values.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+kubevirt:
+  image: registry.suse.com/suse/sles/15.5/virt-operator:1.1.0-150500.8.6.1
+  version: 1.1.0-150500.8.6.1
```
Comment (ipetrov117) [inline charts/kubevirt/0.2.0/values.yaml:3]: If the `1.1.0-150500.8.6.1` is the same for both the `image` and `version`, does it make sense in the K8s resources to refer to the `kubevirt.image` as `{{ .Values.kubevirt.image }}:{{ .Values.kubevirt.version }}`? That way we will not have the same version in two places. WDYT?

Comment (atanasdinov) [reply charts/kubevirt/0.2.0/values.yaml:3]: I was wondering about this. We didn't have the version value in place before so I'm not entirely sure where it is actually used. I guess we can do this given that we're most certainly not going to upgrade again before Edge 3.0 so even if those two are different in a newer version we should be in a good place to address it.

File: charts/kubevirt/0.2.0/Chart.yaml (hunk 4 above)
[the remark this answers is not in the document]
Comment (e-minguez) [reply charts/kubevirt/0.2.0/Chart.yaml]: They keep moving the logos IDK why! 😅

**Comment by ipetrov117** [comment]
Other than the small nit, everything from my POV looks good. I deployed the chart on an rke2 cluster, pods came up and no error logs were present.

File: charts/kubevirt/0.2.0/values.yaml (hunk 5 above)
Comment (atanasdinov) [reply charts/kubevirt/0.2.0/values.yaml:3]: The generated manifests have everything inlined so what I do in this initial step is to grab the most commonly templated values out of the manifests.

I do believe that the suggestion puts us in a better place and I plan on making this much more customisable anyway so it should be a step in the right direction. And if it proves wrong in a later update we can always revert. :)

**Review by ipetrov117 — APPROVED** [review]

**Review by e-minguez — APPROVED** [review]

**Pull request merged**